cli/packages/graphcool-cli-engine/src/Environment.ts

```diff
@@ -656,9 +656,10 @@ Please run ${chalk.bold('graphcool local up')} to start the local cluster.`)
       return this.subscriptionURL({ region, projectId })
     }
 
-    const match = this.clusterEndpoint.match(/.*:(\d+)\/?.*/)
-    const localPort = match ? match[1] : '60000'
-    return this.subscriptionURL({ localPort, projectId })
+    const match = this.clusterEndpoint.match(/https?:\/\/(.*):(\d+)\/?.*/)
+    const localAddr = match ? match[1] : 'localhost'
```

Comment:
This is not going to work, because the regex checks for 'https' to determine between local and cloud. If you have a different IP than localhost, the address will still start with http, not https, so there will be no match.

Reply:
/https?:\/\/(.*):(\d+)\/?.*/   will match both http and https

Reply:
you can try:
 ```javascript
"https://192.168.0.101:60000/system".match(/https?:\/\/(.*):(\d+)\/?.*/)
 "http://192.168.0.101:60000/system".match(/https?:\/\/(.*):(\d+)\/?.*/)
```

Reply:
@kbrandwijk

Reply:
I stand corrected @AlexanderChen1989.